config/deploy.staging.yml

```diff
@@ -44,67 +41,20 @@ builder:
   secrets:
     - RAILS_MASTER_KEY
 
-#
 env:
   clear:
-    RUBY_YJIT_ENABLE: 1
-    RAILS_SERVE_STATIC_FILES: true
```

Comment:
Never used, and static files are now served by default: https://edgeguides.rubyonrails.org/configuring.html#config-public-file-server-enabled